packages/thumbnail/src/spectrum-thumbnail.css

```diff
@@ -12,353 +12,431 @@ governing permissions and limitations under the License.
+                        --mod-thumbnail-focus-indicator-thickness,
+                        var(--spectrum-thumbnail-focus-indicator-thickness)
+                    )
+            ) * -1
     );
+    position: absolute;
 }
-:host([size='l']) {
-    --spectrum-thumbnail-border-color-selected: var(
-        --spectrum-thumbnail-l-border-color-selected,
-        var(--spectrum-alias-border-color-selected)
+:host(.focus-visible) .image-wrapper,
+:host([focused]) .image-wrapper {
+    border-radius: var(
+        --mod-thumbnail-border-radius,
+        var(--spectrum-thumbnail-border-radius)
     );
-    --spectrum-thumbnail-border-size-selected-key-focus: var(
-        --spectrum-thumbnail-l-border-size-selected-key-focus,
-        var(--spectrum-alias-border-size-thick)
+    overflow: hidden;
+}
+:host(:focus-visible) .image-wrapper,
+:host([focused]) .image-wrapper {
+    border-radius: var(
+        --mod-thumbnail-border-radius,
+        var(--spectrum-thumbnail-border-radius)
     );
-    --spectrum-thumbnail-border-size: var(
-        --spectrum-thumbnail-l-border-size,
-        var(--spectrum-alias-border-size-thin)
+    overflow: hidden;
+}
+.spectrum-Thumbnail-layer {
+    align-items: center;
+    border: var(
+            --mod-thumbnail-layer-border-width-outer,
+            var(--spectrum-thumbnail-layer-border-width-outer)
+        )
+        solid
+        var(
+            --mod-thumbnail-layer-border-color-outer,
+            var(--spectrum-thumbnail-layer-border-color-outer)
+        );
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+}
+.spectrum-Thumbnail-layer:before {
+    content: none;
+}
+:host([selected]) .spectrum-Thumbnail-layer {
+    outline: solid;
+    outline-color: var(
+        --highcontrast-thumbnail-border-color-selected,
+        var(
+            --mod-thumbnail-border-color-selected,
+            var(--spectrum-thumbnail-border-color-selected)
+        )
     );
-    --spectrum-thumbnail-border-color: var(
-        --spectrum-thumbnail-l-border-color,
-        var(--spectrum-alias-border-color-translucent)
+    outline-offset: calc(
+        var(
+                --mod-thumbnail-border-width-selected,
+                var(--spectrum-thumbnail-border-width-selected)
+            ) -
+            var(
+                --mod-thumbnail-layer-border-width-inner,
+                var(--spectrum-thumbnail-layer-border-width-inner)
+            )
     );
-    --spectrum-thumbnail-border-color-key-focus: var(
-        --spectrum-thumbnail-l-border-color-key-focus,
-        var(--spectrum-alias-border-color-key-focus)
+    outline-width: var(
+        --mod-thumbnail-border-width-selected,
+        var(--spectrum-thumbnail-border-width-selected)
     );
-    --spectrum-thumbnail-darksquare-background-color: var(
-        --spectrum-thumbnail-l-darksquare-background-color,
-        var(--spectrum-alias-thumbnail-darksquare-background-color)
+}
+.spectrum-Thumbnail-layer-inner {
```

Comment:
This selector not having a conversion points to there being a feature in the CSS implementation that we don't leverage here. What does `layer-inner` do?

Reply:
Code is using it now, `layer-inner` is similar to the `image-wrapper` but for the layer variant.